apps/server/src/modules/board/service/board-do-authorizable.service.ts

```diff
@@ -72,7 +77,7 @@ export class BoardDoAuthorizableService implements AuthorizationLoaderService {
 					userId: user.id,
 					firstName: user.firstName,
 					lastName: user.lastName,
-					roles: [BoardRoles.READER],
+					roles: isDrawing ? [BoardRoles.EDITOR] : [BoardRoles.READER],
```

Comment:
Add comments that it is needed for file updload and link to the element.uc.ts excpetion.

Reply:
done